Phat-Taupe-Pests/BangazonAPI
# Matt product type docs

DESCRIPTION
Added ProductType Model and Controller
ProductType has ability to Get(single or all), Post, Put, and Delete

STEPS TO TEST
1. Clone this branch to a local repo MattProductTypeDocs
2. In terminal, run dotnet build then dotnet run
3. Test functionality
A. Method Post Paste: http://localhost:5000/ProductType into url section on Postman. Paste { "Name": "Lawn and Garden"} into the body section. Send request.
Should return:
{ "ProductTypeID": 1, "Name": "Lawn and Garden" }
B. Method Get(all) Paste: http://localhost:5000/ProductType into url section on Postman. Send request.
Should return:
{ "ProductTypeID": 1, "Name": "Lawn and Garden" }
C. Method Get(specific id) Paste: http://localhost:5000/ProductType/1 into url section on Postman. Send request.
Should return:
{ "ProductTypeID": 1, "Name": "Lawn and Garden" }
D. Method Put Paste: http://localhost:5000/ProductType/1 into url section on Postman. Paste { "ProductTypeID": 1, "Name": "Home Goods"} into the body section. Send request.
Should return:
{ "ProductTypeID": 1, "Name": "Home Goods" }
E. Method Delete Paste: http://localhost:5000/ProductType/1 into url section on Postman. Send Request.
Should Return:
{ "ProductTypeID": 1, "Name": "Home Goods" }

REFERENCE
Ticket - #5
Other - ERD: https://drive.google.com/file/d/0BzCOKyWY4HQpUDU4OG1DcjFLYTA/view
